be/src/runtime/group_commit_mgr.cpp

```diff
@@ -487,31 +487,24 @@ Status LoadBlockQueue::close_wal() {
 
 bool LoadBlockQueue::has_enough_wal_disk_space(
         const std::vector<std::shared_ptr<vectorized::Block>>& blocks, const TUniqueId& load_id,
-        bool is_blocks_contain_all_load_data) {
+        bool is_blocks_contain_all_load_data, size_t content_length) {
     size_t blocks_size = 0;
```

Comment:
It's better to change the function to `bool LoadBlockQueue::has_enough_wal_disk_space(size_t pre_allocated)`